packages/documentation-site/patternfly-docs/content/PatternFly-AI/patternfly-mcp.md

```diff
@@ -0,0 +1,95 @@
+- `fetchDocs`: Used to retrieve the full documentation for a specific component via a URL provided by `usePatternFlyDocs`.
+
+## Get started
+You can install the PatternFly MCP in your preferred development environment. Here are the instructions for popular clients.
+
+### Cursor
+1. While in Cursor, press **cmd** + **shift** +**P**. 
+1. Search for and select **View: Open MCP Settings**. 
+1. At the bottom of the settings menu, click **New MCP Server** to open the `mcp.json` file.
+1. Add the following JSON block within the `mcpServers` object. If `mcpServers` already exists, just add the `patternfly-docs` block inside it.
+
+    ``` {
+    "mcpServers": {
+        "patternfly-docs": {
+        "command": "npx",
+        "args": [
+            "-y",
+            "@patternfly/patternfly-mcp"
+        ],
+        "description": "PatternFly React development rules and documentation"
+        }
+    }
+    }
+    ```
+1. Save and close mcp.json. The MCP Settings will now show "patternfly-docs" as an installed server with its tools enabled.
+<br/ ><br/ >
+	<div class="ws-docs-content-img">
+    ![Cursor's MCP settings, which display patternfly-docs as turned "on".](./img/cursor-mcp-settings.svg)
+    </div>
+    
+### Claude Code
+
+1. From your terminal (before opening Claude Code), run the `claude mcp add` command with your preferred configuration: 
+    - To install the server globally (recommended, so it's available across all your projects), use the `-s user` flag:
+        ```claude mcp add patternfly-mcp -s user -- npx -y @patternfly/patternfly-mcp```
+    - To install for the current project only, run:
+        ```claude mcp add patternfly-mcp -- npx -y @patternfly/patternfly-mcp```
+1. Open Claude Code by typing `claude`.
+1. Inside Claude Code, type the `/mcp` command to view and manage your running MCP servers. You should see `patternfly-mcp` listed as "connected".
+<br/ ><br/ >
+	<div class="ws-docs-content-img">
+    ![Terminal results from the "/mcp" command displaying patternfly-mcp as a connected server.](./img/claude-mcp-servers.svg)
+    </div>
+
+## Example prompts
+Once installed, you can begin asking questions in your chat window and the PatternFly MCP will automatically be referenced for PatternFly-related requests. You can use the PatternFly MCP’s tools for a range of use cases, but here are a few sample scenarios to explore:
+
+### Verifying installation
+- **Prompt:** "Can you see the PF MCP running? If so, what are the tools?"
+- **Expected AI behavior:** The assistant will confirm it sees the PatternFly MCP and list the available tools: mcp_patternfly-docs_usePatternFlyDocs and mcp_patternfly-docs_fetchDocs.
+- **Response:** Yes, the PatternFly MCP is running successfully. There are 2 available tools for you to use: `mcp_patternfly-docs_usePatternFlyDocs` and `mcp_patternfly-docs_fetchDocs`.
+
+### Fetching specific documentation
+- **Prompt:** "Yes, fetch docs for card"
```

Comment:
This was copy/pasted as a direct follow-up to the output from the "verifying installation" prompt above, but we've reframed these here as unrelated sample prompts so I'd suggest rewording this as an independent prompt:

```suggestion
- **Prompt:** "Fetch docs for PatternFly card"
```